fetch.bs

```diff
@@ -2575,6 +2580,7 @@ or a <a for=/>URL</a>.
 
  <li><p>Return <var>location</var>.
 </ol>
+</div>
 
 <p class=note>The <a for=response>location URL</a> algorithm is exclusively used for redirect
```

Comment:
I think it makes semantic sense for this note to be part of the algorithm.

Reply:
Yeah, that makes sense.